mountpoint-s3/src/main.rs

```diff
@@ -13,8 +13,9 @@ use mountpoint_s3::fuse::S3FuseFilesystem;
 use mountpoint_s3::metrics::{metrics_tracing_span_layer, MetricsSink};
 use mountpoint_s3::prefix::Prefix;
 use mountpoint_s3_client::{
-    AddressingStyle, Endpoint, HeadBucketError, ObjectClientError, S3ClientConfig, S3CrtClient,
+    AddressingStyle, Endpoint, HeadBucketError, ImdsCrtClient, ObjectClientError, S3ClientConfig, S3CrtClient,
 };
+
```

Comment:
drop this newline. not sure if there's a good way to enforce this with a style checker instead. :(